### DISCUSSION TITLE
IA Powered terminal

### DISCUSSION THREAD

**Discussion by geanruca** [fish-shell/fish-shell#10099] [Ideas]
Hello! I would like to know if we have any kind of feature with IA to work in Fish using human language. Something like Warp, for example, to ask about commands with context

Thanks!

**Comment by faho** [comment]
No.

We do not support LLMs ("AI"), and if I have anything to say about it we won't. I don't believe in it.

Fundamentally they're awkward heuristics at best and misinformation machines at worst.

And any information they give you depends on the corpus they are trained on. If that corpus is smaller, like for a relatively unknown language like fish script, the output will be worse. I've seen LLMs generate complete nonsense as fish script, for relatively simple questions.

**Reply by geanruca** [reply]
Thanks for the quick answer Faho!
